resources/views/message/index.blade.php

```diff
@@ -51,3 +51,21 @@
         </div>
     </div>
 @endsection
+
+@push('scripts')
+    <script>
+        $(function () {
+            let userId = "{{ auth()->user()->id }}";
+            let ipAddress = '//realtime-app.local';
+            let socketPort = '3001'; // same by port on server.js
+
+            let socket = io(ipAddress + ':' + socketPort, {secure: true, transports: ['websocket']});
```

Comment:
secure: true -> using for SSL
transports: ['websocket'] -> fix "has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource"